docusaurus/docs/create-a-plugin/develop-a-plugin/manage-go.md

```diff
@@ -8,7 +8,7 @@ Develop backend components for a Grafana plugin in [Go](https://go.dev/). The fo
 
 ## Update the Go SDK
 
-To keep the [Grafana plugin SDK for Go](https://grafana.com/docs/grafana/latest/developers/plugins/backend/grafana-plugin-sdk-for-go/) up to date:
+To keep the [Grafana plugin SDK for Go](../../introduction/grafana-plugin-sdk-for-go) up to date:
```

Comment:
Again not a fan of having dependencies on other open PRs. This file isn't live so reviews are trickier.

Reply:
Good point, although in this particular case the SDK file is live as it was merged in https://github.com/grafana/plugin-tools/pull/296.